client/components/review.js

```diff
@@ -125,10 +126,33 @@ class Review extends React.Component {
     );
   }
 
+  changed = () => {
+    if (this.props.changedFromLatest) {
+      return LATEST;
+    }
+    else if (this.props.changedFromGranted) {
+      return GRANTED;
+    }
+  }
+
+  changedBadge = () => {
+    switch (this.changed()) {
+      case LATEST:
+        return <span className="badge changed">changed</span>;
+      case GRANTED:
+        return <span className="badge">amended</span>;
+      default:
+        return null;
+    }
+  }
+
   render() {
     return (
       <div className="review">
         <h3>{this.props.label}</h3>
+        {
+          this.changedBadge()
+        }
         {this.replay()}
```

Comment:
Can we just make the formatting consistent here? Either have the line breaks inside the `{  }` or don't. Either is fine, but formatting the "same" code two different ways right next to each other makes me twitch.